### PR TITLE
Tome radials on AltClick

### PR DESCRIPTION
**Pull request by Allspice**
A whoopsie was done in the other PR #21399 

Tome radials

:cl:
* rscadd: Alt-click tome to make a radial of all talismans in that tome; click one of them to put that in your hand.

### CODE REVIEW

**Comment by DeityLink** [comment]
Nice. Can people see what the talismans do when hovered over?

File: code/datums/gamemode/factions/bloodcult/bloodcult_items.dm [hunk 1]
```diff
@@ -242,6 +236,25 @@ var/list/arcane_tomes = list()
 		else
 			to_chat(user, "<span class='warning'>This tome cannot contain any more talismans. Use or remove some first.</span>")
 
+/obj/item/weapon/tome/AltClick(var/mob/user)
+	var/list/choices = list()
+	var/datum/rune_spell/instance
+	var/list/choice_to_talisman = list()
+	var/image/talisman_image
+	for(var/obj/item/weapon/talisman/T in talismans)
+		talisman_image = new(T.icon, T.icon_state)
```
Comment (Exxion) [inline code/datums/gamemode/factions/bloodcult/bloodcult_items.dm]: You can just do `talisman_image = new(T)` and leave out the following line

**Comment by Allspice** [comment]
@DeityLink The tooltip is set to desc_talisman

**Comment by DeityLink** [comment]
nice

**Pull request merged**